core/cmd/evm_transaction_commands_test.go

```diff
@@ -25,7 +25,8 @@ func TestShell_IndexTransactions(t *testing.T) {
 
 	_, from := cltest.MustAddRandomKeyToKeystore(t, app.KeyStore.Eth())
 
-	tx := cltest.MustInsertConfirmedEthTxWithLegacyAttempt(t, app.TxmStorageService(), 0, 1, from)
+	borm := cltest.NewTxStore(t, app.GetSqlxDB(), app.GetConfig().Database())
```

Comment:
nit: borm was old name when we called the DB interface as ORM.
A better name now is txStore.
rename all borm to txStore.
Also, in case you see txStorageService, that too should be renamed to txStore.

Reply:
Fixed, thanks!